src/components/entity-panel/games/Wows.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div v-if="clickedItem === 'Add'">
+    <TheCreateEntity :clickedItem="clickedItem" :teams="teams" :shipsData="shipsData"/>
+  </div>
+  <div
+    v-else
+  >
+    Other content
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { RoomGetters, RoomState, Ship } from '@/store/modules/room'
+import { Getter } from 'vuex-class'
+import { MenuItem } from '../../TheEntityPanel.vue'
+import { SavedField } from '@/types/Games'
+import { ShipField, ShipItem } from '@/types/Games/Wows'
+import TheCreateEntity from '../sections/TheCreateEntity.vue'
+
+@Component({
+  name: 'WOWS',
+  components: {
+    TheCreateEntity
+  }
+})
+export default class Wows extends Vue {
+  @Prop() private clickedItem!: string;
+  @Prop() private teams!: MenuItem[];
+  @Getter(`room/${RoomGetters.ROOM_STATE}`) roomState!: RoomState;
+
+  get shipsData (): ShipItem[] {
+    const entityPlaceholders: ShipItem[] = [{
+      text: 'Aircraft Carrier (CV)',
+      value: 1,
+      image: this.roomState.game.gameInfo?.ship_type_images.AirCarrier.image,
```

Comment:
use a variable to avoid repeated call to this.roomState.game.gameInfo everytime

Reply:
Sure!